sonar-xml-plugin/src/main/java/org/sonar/plugins/xml/highlighting/XMLHighlighting.java

```diff
@@ -156,8 +156,8 @@ private void highlightStartElement(XMLStreamReader xmlReader, int startOffset) {
   }
 
   private void highlightXmlDeclaration() {
-    if (content.startsWith(XML_DECLARATION_TAG)) {
-      int startOffset = 0;
+    int startOffset = content.startsWith(XmlFile.BOM_CHAR) ? 1 : 0;
+    if (content.startsWith(XML_DECLARATION_TAG, startOffset)) {
```

Comment:
@pynicolas I can't understand why this change is required (as "content" should be the same as before this commit)